jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/AttachHandler.java

```diff
@@ -323,6 +332,88 @@ private boolean initialize() throws IOException {
 		syncFileLock = new FileLock(syncFileObjectTemp.getAbsolutePath(), TargetDirectory.SYNC_FILE_PERMISSIONS);
 		/* give other users read permission to the sync file */
 
+		if (IPC.isLinux) {
+			// Update AttachAPI control file access time every sleepDays (8 days by default).
+			// This is to prevent Linux systemd from deleting the ageing files within /tmp after 10 days.
+			// More details at https://github.com/eclipse-openj9/openj9/issues/18720
+			int sleepDays = 8;
+			String fileAccessUpdateTime = internalProperties.getProperty("com.ibm.tools.attach.fileAccessUpdateTime", "8");
+			IPC.logMessage("fileAccessUpdateTime = " + fileAccessUpdateTime); //$NON-NLS-1$
+			try {
+				sleepDays = new Integer(fileAccessUpdateTime).intValue();  //$NON-NLS-1$
```

Comment:
This should use `Integer.parseInt()` instead.

Reply:
Updated